src/Sarif.Converters.UnitTests/TestData/FortifyFprConverter/Inputs/OneResultBasic.fvdl

```diff
@@ -116,5 +117,17 @@ SqlDataReader objReader = query.ExecuteReader();</Text>
     <Username>johndoe</Username>
     <Platform>Linux</Platform>
   </MachineInfo>
+  <RuleInfo>
+    <Rule id="31D4607A-A3FF-447C-908A-CA2BBE4CE4B7">
+      <MetaInfo>
+        <Group name="package">C Core</Group>
+        <Group name="altcategorySTIG4.8">APSC-DV-002560 CAT I</Group>
```

Comment:
Should these reference IDs make it into the Rule.Properties somewhere?
Is one of these IDs a better choice as the RuleID than the GUID? Which one?

Reply:
I think it should be in rule.guid. I think Michael mentioned this last week.

Reply:
But then I don't know what to use for the ruleId

Reply:
Maybe we could ask Katerina in a thread.
Also, I think I was unclear: I think the secondary IDs (APSC-DV-002560 CAT I; CWE ID 22, CWE ID 73, etc) should also be in the rule in SARIF somewhere, so we know all of those alternative rule identities.

Reply:
Yeah I just noticed I'm not emitting the CWE references. Looking now...

Reply:
OK fixed. I just sent mail to Katrina so I'll wait for a reply before pushing.